Desktop/components/JASP/Widgets/PlotEditor/PlotEditor.qml

```diff
@@ -38,6 +38,7 @@ Popup
 			border.color:	jaspTheme.uiBorder
 			border.width:	preferencesModel.uiScale
 
+			/* This prevents the cursor to get its right shape.
```

Comment:
Sure but now you can no longer click out of the tableview... 
Which is quite normal in my opinion, wouldn't it help to just put the z different?

Reply:
Ok, I fixed it like:
```
Item
					{
						id:					flickChild
						width:				axesFlickable.width
						height:				stack.y + stack.height + jaspTheme.generalAnchorMargin
						
						MouseArea
						{
							id:				focusCatcher	//Because then people can click away from the axistableview or something
							onPressed:		{ forceActiveFocus(); mouse.accepted = false; }
							anchors
							{
								top:		parent.top
								left:		parent.left
								right:		parent.right
							}
							height:			Math.max(flickChild.height, axesFlickable.height)
							z:				-100
						}
```

The flickChild thing is just there to show the location where it ought to be